hera_cal/tests/test_abscal.py

```diff
@@ -1144,6 +1144,68 @@ def test_build_data_wgts(self):
                         else:
                             assert wgts[bl][t, f] == 2
 
+    def test_get_idealized_antpos(self):
+
+        # build 7 element hex with 1 outrigger
+        antpos = hex_array(2, split_core=False, outriggers=0)
+        antpos[7] = np.array([100, 0, 0])
```

Comment:
Add a little more clarification on what you are checking in this test. Something like "ensure that _get_idealised_antpos gives the expected number of dimensions and idealized antennas"

Reply:
Also discuss the test cases below.

Reply:
I've significantly expanded the comments on the unit test